controllers/v1/cloud-services/aws.js

```diff
@@ -0,0 +1,275 @@
+/**
```

Comment:
@Prajwal17Tunerlabs We don't need this file. remove it. Check if we can remove any other code, which is only used by this file

Reply:
removed aws.js file